src/css/styles.css

```diff
@@ -104,6 +109,27 @@ body{
     box-sizing: border-box
 }
 
+.hidden {
+    display: none!important;
+}
```

Comment:
I think if you add the .hidden class selector to the bottom of the css file you won't need the !important.  But of course, this is css we're talking about and so if that would not work for any reason (literally any given css) the !important may be necessary.